### PR TITLE
All shut message

### PR DESCRIPTION
**Pull request by joakim-hove**
Fixes: https://github.com/OPM/opm-simulators/issues/2647

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this please

File: src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp [hunk 1]
```diff
@@ -2824,8 +2795,15 @@ void Schedule::invalidNamePattern( const std::string& namePattern,  std::size_t
         for (const auto& wname : well_names) {
             const auto& well = this->getWell(wname, timeStep);
             const auto& connections = well.getConnections();
-            if (connections.allConnectionsShut())
-                this->updateWellStatus( well.name(), timeStep, Well::Status::SHUT, false);
+            if (connections.allConnectionsShut()) {
+                if (well.getStatus() != Well::Status::SHUT) {
```
Comment (bska) [inline src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp:2799]: If you wanted to, you could fuse these two conditions into one to save one level of nesting.

Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp]: I have updated according to the suggestion.

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**